frontend/src/component/ui/Toast.scss

```
@@ -0,0 +1,33 @@
+@import '../styles/color';
+@import '../styles/corner';
+@import '../styles/font';
+@import '../styles/layouts';
+
+.toast:hover {
+  opacity: 1;
+  cursor: pointer;
+}
+
+.toast-message {
+  font-size: 14px;
+  text-align: center;
+  overflow: hidden;
+  text-overflow: ellipsis;
+
+  @include default-font;
+}
+
+.toast {
+  z-index: 10000;
+  width: 80vw;
+  max-width: 300px;
+  color: #dedede;
+  background-color: $color-dark-gray;
```

```suggestion
  background-color: $color-toast-background;
```
`$color-toast-background` can then reference `$color-dark-gray`.
The goal is to allow us changing the background color globally when retheming.